containers/freesurfer.context/freesurfer.Dockerfile

```diff
@@ -0,0 +1,54 @@
+# docker build for distributing a base fs 7.2.0 container
+
+FROM freesurfer-base as freesurfer
+
+ARG FREESURFER_VERSION
+ENV FREESURFER_VERSION=${FREESURFER_VERSION:-${FREESURFER_VERSION}}
```

Comment:
Revert this. The latter part sets the default if (1) ARG is not defined or (2) then variable is not defined in env.